### DISCUSSION TITLE
Implication of validations with lawfulness warnings

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#560] [Q&A] [answered]
What implication do the validations with lawfulness warnings have in the Production environment?

Will any of them generate manual rejection by the OP?

Will these warning type validations change to error type in the future? Or maybe not?

Thanks in advance,
Alex

**Comment by ferrakaop** [comment, marked as answer]
This is a business question, not directly related to the SDK.

There's some information about lawfulness in the FAQ at https://docs.ted.europa.eu/home/eforms/FAQ/index.html and you can contact the TED Helpdesk if you need more details.

Lawfulness checks in CVS are the only rules that generate warnings (i.e. the don't block validation and submission) and we don't have any immediate plans to change this.
https://docs.ted.europa.eu/eforms/latest/reference/business-rules/index.html#_lawfulness
